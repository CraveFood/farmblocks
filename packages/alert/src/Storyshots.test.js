import initStoryshots from "@storybook/addon-storyshots";

global.window = global;
window.addEventListener = () => {};
window.requestAnimationFrame = () => {
  throw new Error("requestAnimationFrame is not supported in Node");
};

initStoryshots();
