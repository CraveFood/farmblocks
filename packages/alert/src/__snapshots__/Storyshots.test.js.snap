// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Button with some emoji 1`] = `
<button
  className="css-1yjiefr"
  onClick={[Function]}
>
  üòÄ üòé üëç üíØ
</button>
`;

exports[`Storyshots Button with text 1`] = `
<button
  className="css-1yjiefr"
  onClick={[Function]}
>
  Hello Button
</button>
`;

exports[`Storyshots Welcome to Storybook 1`] = `
<article
  className="css-1fqbdip"
>
  <h1
    className="css-nil"
  >
    Welcome to storybook
  </h1>
  <p>
    This is a UI component dev environment for your app.
  </p>
  <p>
    We've added some basic stories inside the
     
    <code
      className="css-mteq83"
    >
      src/stories
    </code>
     
    directory.
    <br />
    A story is a single state of one or more UI components. You can have as many stories as you want.
    <br />
    (Basically a story is like a visual test case.)
  </p>
  <p>
    See these sample
     
    <a
      className="css-ca0824"
      onClick={[Function]}
      role="button"
      tabIndex="0"
    >
      stories
    </a>
     
    for a component called
     
    <code
      className="css-mteq83"
    >
      Button
    </code>
    .
  </p>
  <p>
    Just like that, you can add your own components as stories.
    <br />
    You can also edit those components and see changes right away.
    <br />
    (Try editing the 
    <code
      className="css-mteq83"
    >
      Button
    </code>
     stories located at 
    <code
      className="css-mteq83"
    >
      src/stories/index.js
    </code>
    .)
  </p>
  <p>
    Usually we create stories with smaller UI components in the app.
    <br />
    Have a look at the
     
    <a
      className="css-ca0824"
      href="https://storybook.js.org/basics/writing-stories"
      rel="noopener noreferrer"
      target="_blank"
    >
      Writing Stories
    </a>
     
    section in our documentation.
  </p>
  <p
    className="css-bwdon3"
  >
    <b>
      NOTE:
    </b>
    <br />
    Have a look at the
     
    <code
      className="css-mteq83"
    >
      .storybook/webpack.config.js
    </code>
     
    to add webpack loaders and plugins you are using in this project.
  </p>
</article>
`;
